rust/connlib/snownet/src/node.rs

```diff
@@ -1815,3 +1812,13 @@ where
 
     Some(transmit)
 }
+
+fn new_agent() -> IceAgent {
+    let mut agent = IceAgent::new();
+    agent.set_max_candidate_pairs(300);
+    agent.set_timing_advance(Duration::ZERO);
+    agent.set_max_stun_retransmits(8);
+    agent.set_max_stun_rto(Duration::from_millis(1500));
```

Comment:
These are the settings we also currently use in our fork. See first commit in https://github.com/algesten/str0m/compare/main...firezone:str0m:main.

We may want to tweak these eventually but for the purpose of this PR, I'd like to keep them as is and only focus on the dependency upgrade.